webots_ros2_epuck/worlds/rats_life_benchmark.wbt

```diff
@@ -1,4 +1,14 @@
-#VRML_SIM R2022a utf8
+#VRML_SIM R2022b utf8
+
+EXTERNPROTO "webots://projects/objects/backgrounds/protos/TexturedBackground.proto"
+EXTERNPROTO "webots://projects/objects/backgrounds/protos/TexturedBackgroundLight.proto"
+EXTERNPROTO "../protos/LegoTallWall.proto"
+EXTERNPROTO "../protos/LegoTallInterval.proto"
```

Comment:
Is there a reason why these protos have a relative path?

Reply:
because they reference PROTO exclusive to the ros2 repo: https://github.com/cyberbotics/webots_ros2/tree/master/webots_ros2_epuck/protos